bioconda_utils/utils.py

```diff
@@ -1213,6 +1221,13 @@ def set_cache(self, cache):
         else:
             self.cache_file = cache
 
+    @property
+    def channels(self):
```

Comment:
I'd do this test in `__init__()`, rather than adding things that "fail late" in a property.